events_structs.go

```diff
@@ -818,6 +819,21 @@ func (ev *EventDeleteAutoscalerType) String() string {
 	return fmt.Sprintf("Alert deleted about %s on container %s", d.Metric, d.ContainerType)
 }
 
+type EventStartRegionMigrationTypeData struct {
+	MigrationID string `json:"migration_id"`
+	Destination string `json:"destination"`
+	DstAppName  string `json:"dst_app_name"`
+}
+
+type EventStartRegionMigrationType struct {
+	Event
+	TypeData EventStartRegionMigrationTypeData `json:"type_data"`
+}
+
+func (ev *EventStartRegionMigrationType) String() string {
+	return fmt.Sprintf("Application region migration started to %s/%s", ev.TypeData.Destination, ev.TypeData.DstAppName)
```

Comment:
It would be great to also include the source of the migration

Reply:
You know the source it's the app you asked the timeline for.

Reply:
We never include the source of the events in those messages.

Reply:
It's not necessarily obvious. If I type:

```
scalingo -a biniou timeline
```

What is the source?

Reply:
biniou ?

Reply:
No, I mean the source region. It's probably `agora-fr1` for instance.

Reply:
The perfect message for me would be something like:

```
Application region migration from agora-fr1 to osc-fr1/biniou started

Reply:
Oh ... It's not that easy. I'll check how to do that.